src/Raven.Pal/Raven.Pal.vcxproj

```diff
@@ -46,7 +46,7 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'" Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
-    <UseDebugLibraries>false</UseDebugLibraries>
+    <UseDebugLibraries>true</UseDebugLibraries>
```

Comment:
So in `Release|x64` we use debug libraries? (not sure what does exactly mean though)

Reply:
That is used _only_ when I debug this - the actual building process is not related to those values. 
Can safely ignore this.